examples/acoustic_example.py

```diff
@@ -106,21 +106,26 @@ def damp_boundary(damp, h, nbpml):
             damp[:, :, -(i + 1)] += val
 
 nrec, _ = data.traces.shape
-m = DenseData("m", model.vp.shape, dtype)
+m = DenseData(name="m", shape=model.vp.shape, dtype=dtype)
 m.data[:] = model.vp**(-2)
-u = TimeData("u", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
-v = TimeData("v", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
+u = TimeData(name="u", shape=m.shape, time_dim=nt, time_order=t_order, save=False, dtype=m.dtype)
```

Comment:
I am essentially talking about the fact that passing a 2-dimensional shape leads to an object that is 3 dimensional in memory. The size of the 3rd dimension is an implementation detail that we should definitely hide from the user, I agree.